src/main/java/com/aws/greengrass/clientdevices/auth/CertificateManager.java

```diff
@@ -70,6 +74,10 @@ public void updateCertificatesConfiguration(CertificatesConfig certificatesConfi
         this.certificatesConfig = certificatesConfig;
     }
 
+    public void setCAConfiguration(CAConfiguration caConfiguration) {
```

Comment:
Right now you have `setCAConfiguration` and `updateCAConfiguration` -- remove one of them